### DISCUSSION TITLE
Will GHelper Work On The New ROG STRIX X870E-E?

### DISCUSSION THREAD

**Discussion by DeathStalker77** [seerge/g-helper#3447] [Q&A] [answered]
Greetings all!
I just got this board for my new build, but I won't be doing the build for a couple weeks.  Has this board been requested for support yet?

Thanks!

--- DS

**Comment by seerge** [comment, marked as answer]
@DeathStalker77 hello, if it’s a desktop motherboard - then answer is no. It works only on asus laptops or portable devices (ROG Ally).

**Reply by DeathStalker77** [reply]
Thanks.  Any chance you might expand it to cover mobos?  **AC** is sadly such garbage :-(

--- DS

**Reply by seerge** [reply]
@DeathStalker77 hello, AC for laptops and for desktops are practically 2 separate apps under same name-umbrella. So there is no chances G-Helper will be working on motherboards.

If you just need it for backlight controls, you can try OpenRGB or SignalRGB apps. They work quite well.

For fans you can try https://github.com/Rem0o/FanControl.Releases
